src/navigation/screenOptions.tsx

```diff
@@ -45,7 +45,7 @@ export const getScreenOptions =
     return {
       // header gradient:
       // https://stackoverflow.com/questions/44924323/react-navigation-gradient-color-for-header
-      headerBackground: () => <DiagonalGradient />,
+      headerBackground: () => <DiagonalGradient colors={[colors.surface, colors.surface]} />,
```

Comment:
please remove the diagonal gradient entirely here, as we habe one color to show now

Reply:
ok, right. can happen. what if we make a condition on the array? if the two color values are different, we use gradient, otherwise just a simple background color? i would like to avoid rendering a gradient component if we do not want to have gradient.